### ISSUE TITLE
components version change 

### ISSUE THREAD

**Issue by productdevbook** [oku-ui/docs#9]
https://github.com/oku-ui/docs/assets/38668796/8c37d1ac-17de-4ad0-b9ca-409fd3f436eb

Here I want to change the version. It's like in Radix but I couldn't. How can we print this with nuxt-content.

branch: [version](https://github.com/oku-ui/docs/tree/version)

**### TODO**

- [x] When I click the label, the md content of the latest version should be added automatically. [radix check docs](https://www.radix-ui.com/docs/primitives/components/label)
- [x] when i navigate to the versions it should only show the contents of the versions. And router change xxxx..'/label/${version}`
- [x] click version sidebar dont selected problem

in short it should be the same as radix
